src/ralph/security/migrations/0007_auto_20170331_0934.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
No reason IIRC except personal preferences actually. Let's leave it as is it then.